### PR TITLE
update to ruby 3.0 and rails 6.1

### PR DESCRIPTION
**Pull request by soberstadt**

### CODE REVIEW

**Comment by soberstadt** [comment]
I QAed this on https://text-steps-widget-stage.cru.org/step_set_subscriptions/new

File: config/environments/test.rb [hunk 1]
```diff
@@ -18,7 +18,7 @@
   # Configure public file server for tests with Cache-Control for performance.
   config.public_file_server.enabled = true
   config.public_file_server.headers = {
-    "Cache-Control" => "public, max-age=#{1.hour.to_i}"
+    "Cache-Control" => "public, max-age=3600"
```
Comment (jbirdjavi) [inline config/environments/test.rb:21]: What's the reason for this change?

**Review by jbirdjavi — CHANGES_REQUESTED** [review]
I might try to pull in a few more changes from https://railsdiff.org/6.0.3.6/6.1.3.1, but the one that sticks out the most is `config.load_defaults 6.1` in `config/application.rb`.

File: config/environments/test.rb (hunk 1 above)
Comment (soberstadt) [reply config/environments/test.rb:21]: It seems to execute this before the active support helpers are loaded allowing this to work. It didn't seem like a big deal to hard-code it.

Comment (jbirdjavi) [reply config/environments/test.rb:21]: Interesting.  The default hasn't changed, so I'm not sure why it wouldn't work, but also not a big deal.  :)

Comment (soberstadt) [reply config/environments/test.rb]: I was able to revert this because of https://github.com/CruGlobal/text-steps-widget/pull/6/files#diff-77f322f5ab0c54b1f2793b339574333dc55889645d910a93ede0fd7aa13b217aR3

**Review by jbirdjavi — APPROVED** [review]
